hols/todo/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Remove this script, replace it with an aggregator pom and README instructions

Reply:
The aggregator is already there, I'll remove the build script

Reply:
done